docs/modules/ROOT/pages/references/functional-requirements/portal.adoc

```diff
@@ -24,7 +24,7 @@ This page discusses functional requirements targeting the Control API and the {p
 * Create Organization
 * List Organizations
 * Edit Organization Metadata
-* Invite User to Organization
+* Invitation User to Organization
```

Comment:
Do you intentionally use "invitation" as a verb throughout this document? The bullet point sounds weird when I read it.

Reply:
I was too overzealous with search/replace 🤦‍♀️

Reply:
Thanks for pointing this out 😊